aws-ecs/bastion.userdata.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -x
+
+# updates
+yum update -y
+amazon-linux-extras install postgresql10
```

Comment:
Can we use postgresql14? Is there such a package?

Reply:
This is for the postgresql client, right?

Reply:
Done